### PR TITLE
Consumer API: Rename "IntegrationEvent" to "DomainEvent"

### PR DESCRIPTION
**Pull request by MH321Productions**
# Readiness checklist

-   [ ] I added/updated unit tests.
-   [ ] I added/updated integration tests.
-   [x] I ensured that the PR title is good enough for the changelog.
-   [x] I labeled the PR.

### CODE REVIEW

File: Modules/Devices/src/Devices.Application/Extensions/IServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -36,10 +36,11 @@ private static void AddEventHandlers(IServiceCollection services)
 
     private static IEnumerable<Type> GetAllIntegrationEventHandlers()
     {
-        var integrationEventHandlerTypes = from t in Assembly.GetExecutingAssembly().GetTypes()
-                                           from i in t.GetInterfaces()
-                                           where t.IsClass && !t.IsAbstract && i.IsGenericType && i.GetGenericTypeDefinition() == typeof(IIntegrationEventHandler<>)
-                                           select t;
+        var integrationEventHandlerTypes =
```
Comment (tnotheis) [inline Modules/Devices/src/Devices.Application/Extensions/IServiceCollectionExtensions.cs:39]: rename

File: Modules/Quotas/src/Quotas.Application/Extensions/IEventBusExtensions.cs [hunk 2]
```diff
@@ -11,7 +11,17 @@
 using Backbone.Modules.Quotas.Application.IntegrationEvents.Incoming.TierOfIdentityChanged;
 using Backbone.Modules.Quotas.Application.IntegrationEvents.Incoming.TierQuotaDefinitionDeleted;
```
Comment (tnotheis) [inline Modules/Quotas/src/Quotas.Application/Extensions/IEventBusExtensions.cs:12]: Why is there still the "IntegrationEvents" namespace?

File: Modules/Quotas/src/Quotas.Application/Extensions/IServiceCollectionExtensions.cs [hunk 3]
```diff
@@ -48,12 +48,12 @@ private static void AddMetricCalculators(this IServiceCollection services)
 
     private static IEnumerable<Type> GetAllIntegrationEventHandlers()
     {
-        var integrationEventHandlerTypes = from t in Assembly.GetExecutingAssembly().GetTypes()
-                                           from i in t.GetInterfaces()
-                                           where t.IsClass && !t.IsAbstract && i.IsGenericType && i.GetGenericTypeDefinition() == typeof(IIntegrationEventHandler<>)
-                                           select t;
+        var integrationEventHandlerTypes =
```
Comment (tnotheis) [inline Modules/Quotas/src/Quotas.Application/Extensions/IServiceCollectionExtensions.cs:51]: rename

**Review by tnotheis — CHANGES_REQUESTED** [review]

**Comment by MH321Productions** [comment]
I now performed a global find/replace in the files, as well as one for the file and directory names

**Review by tnotheis — APPROVED** [review]

**Pull request merged**